docs/DESIGN.md

```diff
@@ -43,36 +43,29 @@ The following sections outline the identified use cases and stories per componen
 
 We propose that there should be a single CRD for Thanos Receive.
 
-_ThanosReceive_ will be responsible for managing the lifecycle Receive routing and ingest components.
-It will manage the hashring configuration for the ingest components that it deploys.
+*ThanosReceive* will be responsible for managing the lifecycle Receive routing and ingest components. It will manage the hashring configuration for the ingest components that it deploys.
```

Comment:
```suggestion
*ThanosReceive* will be responsible for managing the lifecycle of Receive routing and ingest components. It will manage the hashring configuration for the ingest components that it deploys.
```